dockerizing-notes.md

```diff
@@ -19,6 +20,7 @@
 
 ## TODO:
 
-* figure out what is needed for also developing `cg-style` (team usually uses `npm link`)
 * figure out why the shrinkwrap doesn't work -- looks like it has to do
```

Comment:
Oh, um, actually I am not sure. I thought I saw the error on my system, where `npm install` failed because of that one `fsevents` darwin dependency, and then I re-ran the thing with `--no-shrinkwrap` or whatever and it worked. But maybe I had accidentally removed some stuff that I thought was `nvm` related but was actually related to your shrinkwrap fix?

One frustrating thing right now is that it's cumbersome to test this because `npm install` takes so long, especially when it needs to be run *twice* (once in each repo). 😞